core/execution/future/market.go

```diff
@@ -1702,6 +1702,19 @@ func (m *Market) validateOrder(ctx context.Context, order *types.Order) (err err
 		}
 	}
 
+	if order.OriginalPrice != nil {
+		return m.validateTickSize(order.OriginalPrice)
+	}
+
+	return nil
+}
+
+func (m *Market) validateTickSize(price *num.Uint) error {
+	d := num.UintZero().Div(price, m.mkt.TickSize)
+	d = num.UintZero().Mul(d, m.mkt.TickSize)
```

Comment:
Why not just `.Mod()`? The code is the same as `.Div()` except it returns the remainder and we can then save ourselves a multiply?